new-log-viewer/src/services/MainWorker.ts

```diff
@@ -41,6 +43,18 @@ onmessage = async (ev: MessageEvent<MainWorkerReqMessage>) => {
 
     try {
         switch (code) {
+            case WORKER_REQ_CODE.EXPORT_LOG: {
```

Comment:
Why do we include `decoderOptions` into EXPORT_LOG's arguments list but leave it unused?